spec/rspec/rails/example/view_example_group_spec.rb

```diff
@@ -227,13 +237,13 @@ def controller
         with_isolated_config do
           run_count = 0
           RSpec.configuration.before(:each, :type => :view) do
-            allow(view).to receive(:a_stubbed_helper) { :value }
+            allow(view).to receive(:render) { :value }
             run_count += 1
           end
           group = RSpec::Core::ExampleGroup.describe 'a view', :type => :view do
             specify { true }
           end
-          group.run NullObject.new
+          group.run
```

Comment:
Doesn't this cause the groups output to be output to the real formatter?

Reply:
There's no difference in the output.

Reply:
I was surprised by that assertion, as the point of this object is to prevent the real reporter being used, but its failing locally for me? I want to investigate this further as its not possible to prove either way on CI without changing the format...

What I expect this to do is to add the spec you've specified above into the reporter, so if you run this spec on its own you'd see "2 examples passed" for this one example.

Reply:
If I run it focused (`rspec spec/rspec/rails/example/view_example_group_spec.rb:236`), it fails locally. However, the spec when run altogether passes.
This weird behavior remains even with just one example left, and if you remove `module` namespace, and `with_isolated_`, and nesting contexts. I've put it on my list to investigate.

Reply:
Maybe it doesn't add to the main reporter because of https://github.com/rspec/rspec-rails/pull/2264/files#diff-93830fa29d616f7c87903d08b5b1b29aR55 ?
```ruby
  config.around(:example) do |example|
    real_world = RSpec.world
    RSpec.instance_variable_set(:@world, RSpec::Core::World.new)
    example.run
    RSpec.instance_variable_set(:@world, real_world)
  end
```

Reply:
I'm not sure if `with_isolated_config` is even necessary in specs with that hook.

Reply:
So I got to the bottom of why this doesn't run when focused. If you're interested its an artefact in our filtering, `with_isolated_config` does not remove filters from this spec, as they exist in the world and the world is not isolated. That is something I might consider fixing at some point but is waaaaaaaaay out of scope for this.

I also confirmed this removal is ok, we now use a `NullReporter` by default, a change which post-dates this spec I suspect.